js/app.js

```diff
@@ -1,29 +1,181 @@
-      textEditor.style.display = "none";
-      textDisplay.style.display = "block";
-      isEditing = false;
+  const movies = await getMovies(title, type, page);
+  displayResults(movies);
+});
+
+/* waiting for response with the movie data of the specified page */
+async function getMovies(title, type, page) {
+  const url = `https://www.omdbapi.com/?s=${encodeURIComponent(
+    title
+  )}&type=${encodeURIComponent(type)}&page=${page}&apikey=${apiKey}`;
+
+  const response = await fetch(url);
+  const data = await response.json();
+  const totalResults = data.totalResults;
+
+  console.log(data);
+
+  if (data.Response === "True") {
+    displayPagination(totalResults, title, type, page);
+
+    return data.Search;
+  } else {
+    return [];
+  }
+}
+
+/* displaying number of pages and controls */
+function displayPagination(totalResults, title, type, page) {
+  let currentPage = page;
+  let totalPages = Math.ceil(totalResults / 10);
+
+  if (totalPages > 100) {
+    totalPages = 100;
+  }
+
+  paginationContainer.style.display = "flex";
+  paginationContainer.innerHTML = `
+      <button class="previous-button">previous page</button>    
+      <p>Page ${currentPage} of ${totalPages}</p>
+      <button class="next-button">next page</button>
+  `;
+
+  const previousButton = document.querySelector(".previous-button");
+  const nextButton = document.querySelector(".next-button");
+
+  if (currentPage === 1) {
+    previousButton.disabled = true;
+  } else {
+    previousButton.disabled = false;
+    previousButton.addEventListener("click", async (event) => {
+      currentPage--;
+
+      document.querySelector("#search-form").scrollIntoView({
+        behavior: "smooth",
+      }); /* focus on the top of the page */
+
+      const movies = await getMovies(title, type, currentPage);
+      displayResults(movies);
+    });
+  }
+
+  if (currentPage === 100 || currentPage === totalPages) {
+    nextButton.disabled = true;
+  } else {
+    nextButton.disabled = false;
+
+    nextButton.addEventListener("click", async (event) => {
+      currentPage++;
+
+      document.querySelector("#search-form").scrollIntoView({
+        behavior: "smooth",
+      }); /* focus on the top of the page */
+
+      const movies = await getMovies(title, type, currentPage);
+      displayResults(movies);
+    });
+  }
+
+  console.log(`totalResults: ${totalResults}`);
+  console.log(`currentPage: ${page}`);
+  console.log(`totalPages: ${totalPages}`);
+}
+
+/* displaying movie data and buttons for details */
+function displayResults(movies) {
+  resultsContainer.innerHTML = "";
+
+  if (movies.length === 0) {
+    resultsContainer.innerHTML = `
+        <h1>Movie not found!</h1>`;
+    return;
+  }
+
+  for (const movie of movies) {
+    const div = document.createElement("div");
+
+    let realPoster;
+    if (movie.Poster === "N/A") {
+      realPoster = "./img/no_image.jpg";
+    } else {
+      realPoster = movie.Poster;
     }
```

Comment:
a good place for ternary

Reply:
realPoster =
    movie.Poster === "N/A"
      ? (realPoster = "./img/no_image.jpg")
      : (realPoster = movie.Poster);